SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -85,7 +84,9 @@ void FSpatialGDKEditorLayoutDetails::CustomizeDetails(IDetailLayoutBuilder& Deta
 		[
 			SNew(SButton)
 			.VAlign(VAlign_Center)
-			.OnClicked(this, &FSpatialGDKEditorLayoutDetails::GenerateDevAuthToken)
+			.OnClicked(FOnClicked::CreateLambda([] {
+				return FSpatialGDKEditorCommandLineArgsManager::GenerateDevAuthToken();
+			}))
```

Comment:
Replace the lambdas with
```suggestion
			.OnClicked_Static(FSpatialGDKEditorCommandLineArgsManager::GenerateDevAuthToken)
```